x-pack/legacy/plugins/uptime/public/pages/404.tsx

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import {
+  EuiEmptyPrompt,
+  EuiPanel,
+  EuiTitle,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+export const NotFoundPage = () => (
+  <EuiFlexGroup justifyContent="center">
+    <EuiFlexItem grow={false}>
+      <EuiPanel>
+        <EuiEmptyPrompt
+          iconType="faceNeutral"
+          iconColor="subdued"
+          title={
+            <EuiTitle size="m">
+              {
+                <h3>
+                  {i18n.translate('xpack.uptime.emptyStateError.404', {
```

Comment:
We can use [`<FormattedMessage>`](https://github.com/elastic/kibana/tree/master/packages/kbn-i18n#react) in this case. We'll end up with something like:

```typescript
<EuiTitle>
  <h3>
    <FormattedMessage
      defaultMessage="Page not found"
      id="xpack.uptime.emptyStateError.404"
    />
  </h3>
</EuiTitle>
```

In any case, also change the string to only capitalize the first word, like `'Page not found'`.

Additionally, let's give a better ID. Something like `xpack.uptime.notFoundPage.title` is more appropriate.